### ISSUE TITLE
Add group of stacked bar chart to profile section on pages

### ISSUE THREAD

**Issue by rapee** [codeforthailand/politician-directory#68]
Prerequisite: #66

Add <StackedBarChart /> component to Profile section for many pages (Party, Cabinet, Senate, House of Rep.) to display:

- Type of Senate (โดยตำแหน่ง / สรรหา / โดย คสช. ) (Only Senate page)
- Type of MP (สส. บัญชีรายชื่อ / แบ่งเขต) (Only House of Rep. page)
- Gender
- Age range
- Education
- Occupation group (showing top 3 and the rest)

You need to bind GraphQL data (count of each category) in order to inject it to components

<img width="1090" alt="Screen Shot 2019-11-03 at 15 44 18" src="https://user-images.githubusercontent.com/924635/68082877-9ad6ff00-fe54-11e9-9d9f-80b368f6f80e.png">

**Comment by rapee** [comment]
Duplicate of #71
